src/site/content/en/blog/2022-state-of-css/index.md

```diff
@@ -0,0 +1,2008 @@
+- web.dev: [/accent-color/](/accent-color/)   
+- Post by [Bramus](https://twitter.com/bramus): [Tint User-Interface Controls
+  with CSS
+  accent-color](https://www.bram.us/2021/08/23/tint-user-interface-controls-with-css-accent-color/)
+
+### Color level 4 and 5
+
+The web has been dominated by sRGB for the past decades, but in an expanding
+digital world of high-definition displays and mobile devices pre-equipped with
+OLED or QLED screens, sRGB is not enough. Furthermore, dynamic pages that adapt
+to user preferences are expected, and color management has been a growing
+concern for designers, design systems, and code maintainers. 
+
+Not in 2022 though—CSS has a number of new color functions and spaces:
+- Colors that reach into the HD color capabilities of displays.
+- Color spaces that match an intent, such as perceptual uniformity.
+- Color spaces for gradients that drastically change the interpolation outcomes.
+- Color functions to help you mix and contrast, and choose which space you do the work in.
+
+Before all these color features, design systems needed to precalculate proper
+contrasting colors, and ensure appropriately vibrant palettes, all while
+preprocessors or JavaScript did the heavy lifting. 
+
+After all these color features, the browser and CSS can do all the work,
+dynamically and just in time. Instead of sending many KBs of CSS and JavaScript
+to users to enable theming and data visualization colors, CSS can do the
+orchestrating and calculations. CSS is also better equipped to check for support
+before usage or handle fallbacks gracefully.
+
+```css
+@media (dynamic-range: high) {
+  .neon-pink {
+    --neon-glow: color(display-p3 1 0 1);
+  }
+}
+
+@supports (color: lab(0% 0 0)) {
+  .neon-pink {
+    --neon-glow: lab(150% 160 0);
+  }
+}
+```
+
+#### `hwb()`
+
+{% BrowserCompat 'css.types.color.hwb' %}
+
+HWB stands for hue, whiteness, and blackness. It presents itself as a
+human-friendly way of articulating color, as it's just a hue and an amount of
+white or black to lighten or darken. Artists who mix colors with white or black
+may find themselves appreciating this color syntax addition.
+
+{% Codepen {
+ user: 'web-dot-dev',
+ id: 'KKZdeep',
+ height: 500,
+ tab: 'css,result'
+} %}
+
+Using this color function results in colors from the sRGB color space, the same
+as HSL and RGB. In terms of newness for 2022, this doesn’t give you new colors,
+but it may make some tasks easier for fans of the syntax and mental model.
+
+##### Resources
+
+- Spec:
+  [https://www.w3.org/TR/css-color-4/#the-hwb-notation](https://www.w3.org/TR/css-color-4/#the-hwb-notation)   
+- MDN:
+  [https://developer.mozilla.org/docs/Web/CSS/color_value/hwb()](https://developer.mozilla.org/docs/Web/CSS/color_value/hwb())  
+- Post by [Bramus](https://twitter.com/bramus): [hwb() – a color notation for
+  humans?](https://www.bram.us/2021/12/30/hwb-a-color-notation-for-humans/)
```

Comment:
Bramus was referencing this post https://www.stefanjudis.com/blog/hwb-a-color-notation-for-humans/ with his post, so this would be the better place to link, I think.